### DISCUSSION TITLE
Welcome to ArchEclipse Discussions!

### DISCUSSION THREAD

**Discussion by AymanLyesri** [AymanLyesri/ArchEclipse#54] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Any Suggestions or new Ideas are Welcome**

**Comment by sanvyx-sancre** [comment]
The wallpapers got me har i mean got me thinking hard, i tought "Should i get into art again" haha

**Comment by mrvictor22** [comment]
If i need to change on the workspace 4, from use spotify to another music client, where is the config file or configs files thas shuld I need to modify, also I will use Cider besides spotify.

**Reply by mrvictor22** [reply]
> If i need to change on the workspace 4, from use spotify to another music client, where is the config file or configs files thas shuld I need to modify, also I will use Cider besides spotify.

@AymanLyesri

**Reply by mrvictor22** [reply]
Spanish Version:
---

### 💡 [TIP] Cider en Workspace 4 usando Flatpak en Hyperland

Si estás empezando con **Hyperland** y estás usando  ArchEclipse, puede que te topes con este problema: modificás `workspaces.conf` para lanzar una app al inicio en un workspace específico… ¡pero no pasa nada! 😤

En mi caso, quería que **Cider** (instalado por Flatpak) se abriera automáticamente en el workspace 4. Hice esto:

configs/custom/workspaces.conf
`workspace = 4, gapsout:69, on-created-empty:flatpak run sh.cider.genten`


**¡Pero no funcionaba!** Luego de varios intentos, descubrí que el rice no aplicaba los cambios automáticamente.

✅ **Solución**:  
Corrí el script de actualización del rice vía terminal, y luego de eso, ¡todo funcionó perfectamente!

---

🔧 Este tip es para la gente nueva en Hyperland:  
A veces el problema no es tu sintaxis, sino que tu entorno simplemente **no está cargando tus cambios**. Siempre que edites archivos `.conf`, reiniciá Hyprland o corré los scripts de actualización del rice si los hay.

---

English Version:

### 💡 [TIP] Automatically Launch Flatpak App (Cider) on a Specific Workspace in Hyperland

If you're new to **Hyperland** and using a preconfigured rice (like **ArchEclipse**), you might run into this issue:

You edit your `workspaces.conf` file to launch an app automatically on a specific workspace… but **nothing happens**. 😩

In my case, I wanted **Cider** (installed via Flatpak) to launch on workspace 4. My config looked like this:

configs/custom/workspaces.conf
`workspace = 4, gapsout:69, on-created-empty:flatpak run sh.cider.genten`


Everything looked fine, **but it didn’t work**. After several retries, I realized the rice wasn't applying my updated configs.

✅ **The fix**:  
I simply ran the rice update script in the terminal (or whatever custom updater the rice provides), and then it worked perfectly.

---

🔧 **Tip for Hyperland newcomers**:  
Sometimes your config syntax is correct — the issue is just that **your environment isn't loading the new config**. Always reload Hyperland (`hyprctl reload`) or re-run the rice setup/update script after changes.
